src/main/java/com/qa/opencart/pages/LoginPage.java

```diff
@@ -40,6 +41,10 @@ public String getLoginPageUrl(){
         return url;
     }
 
+    public void isForgotPasswordLinkExist(){
```

Comment:
Since this is dummy approving this.
Make sure to at least add comments when adding any code